### ISSUE TITLE
Data Ajax: Not working in Basic HTML mode

### ISSUE THREAD

**Issue by shawnthompson** [wet-boew/wet-boew#8419]
When you turn on basic HTML mode, ajax content do not get pulled in.

http://wet-boew.github.io/wet-boew/demos/data-ajax/data-ajax-en.html?wbdisable=true

**Comment by thekodester** [comment]
Not sure if there a simple solution since AJAX requires JS and Basic Mode is basically switching it off.

Also AJAX tables don't display data (filtering example): http://wet-boew.github.io/v4.0-ci/demos/tables/tables-en.html?wbdisable=true

So could be expected behaviour.

**Comment by shawnthompson** [comment]
It used to work, not working now. I couldn't find any talk about it's behaviour being changed in any commits or issues. Although I'm horrible at searching so I might have missed it.

**Comment by duboisp** [comment]
@shawnthompson I don't recall to have seen data-ajax working in basic HTML mode.

But since the [Design Decision 4](https://wet-boew.github.io/wet-boew-documentation/decision/4.html) has been approved at the last WET Roadmap meeting which provide more clarification about the behaviour for Basic HTML mode. I think it is safe that we look a way to let that plugin run even in Basic HTML mode. 

We should think about an long term to allow some plugin to init in basic HTML mode. What do you think of having it's own init event like ```wb.init.basic``` or ```wb.initbasic```

Feel free to submit a PR that would do it.

**Comment by shawnthompson** [comment]
You're right, I testing version 4.0.27 and it didn't work either. This is a mistake on our end. Sorry.

Although I do agree that we should let this plugin run in _basic mode_. 

I don't have time to submit a PR for that but maybe @StdGit will when he comes back from vacation.
